src/core/object/symbol.rs

```diff
@@ -54,6 +58,13 @@ impl SymbolName {
             SymbolName::Uninterned(x) => x.get().as_bytes(),
         }
     }
+
+    fn as_str(&self) -> &str {
```

Comment:
We already have this functionality with the `name` method:

https://github.com/CeleritasCelery/rune/blob/890466266c751816360a2bf3136312943d4eca36/src/core/object/symbol.rs#L298-L303